.golangci.yml

```diff
@@ -0,0 +1,40 @@
+---
+linters:
+  enable-all: true
+  disable:
+    - exportloopref # WARN The linter 'exportloopref' is deprecated (since v1.60.2) due to: Since Go1.22 (loopvar) this linter is no longer relevant. Replaced by copyloopvar.
+    - err113 # WARN [lintersdb] The name "goerr113" is deprecated. The linter has been renamed to: err113
+    - tagliatelle
+    - wsl
+    - nlreturn
+    - lll
+    - godot
+    - varnamelen
+    - ireturn
+    - depguard
+    - tagalign
+    # TODO
+    - dupl
+    - wrapcheck
+    - testpackage
+    - godox
+    - nestif
+    - funlen
+    - gocognit
+    - gocyclo
+    - exhaustruct
+    - dupword
+    - musttag
+    - forbidigo
+    # tentative
+    - revive
+    - unparam
+    - forcetypeassert
+    - noctx
+    - stylecheck
+    - perfsprint
+    - prealloc
```

Comment:
Currently I disabled following linters for reducing complexity of this PR.
We should re-enable them in the future.